tests/conftest.py

```diff
@@ -822,6 +823,14 @@ def cluster_teardown_finalizer():
     if deploy:
         # Deploy cluster
         deployer.deploy_cluster(log_cli_level)
+        # Workaround for #1777384 - enable container_use_cephfs on RHEL workers
+        ocp_obj = ocp.OCP()
+        cmd = ['/usr/sbin/setsebool -P container_use_cephfs on']
+        workers = get_typed_worker_nodes(os_id="rhel")
+        for worker in workers:
+            log.info(f"{worker} is a RHEL based worker - applying '{cmd}'")
+            node = worker.get().get('metadata').get('name')
+            ocp_obj.exec_oc_debug_cmd(node=node, cmd_list=cmd)
```

Comment:
@gklein ,
Would you like to use https://github.com/red-hat-storage/ocs-ci/blob/c106a8c69652128f3cfcaf20866679f602ccf524/ocs_ci/ocs/resources/pod.py#L343 instead of exec_oc_debug_cmd ?

Reply:
Yes, it might be better in this case. I just wonder how to get the ssh private key . It's  look like I need to handle this part myself.